src/StackExchange.Redis/Interfaces/IDatabase.cs

```diff
@@ -472,6 +472,18 @@ public interface IDatabase : IRedis, IDatabaseAsync
         /// <returns>The endpoint serving the key.</returns>
         EndPoint IdentifyEndpoint(RedisKey key = default(RedisKey), CommandFlags flags = CommandFlags.None);
 
+        /// <summary>
+        /// Copy key from the selected key to the specified destination key.
+        /// </summary>
+        /// <param name="source">Copy the value stored at the source.</param>
+        /// <param name="destination">Copy the value to destination in the logical database used by the connection.</param>
+        /// <param name="database">Specifying an alternative logical database index for the destination key.</param>
+        /// <param name="replace">Removes the destination key before copying the value to it.</param>
+        /// <param name="flags">The flags to use for this operation.</param>
+        /// <returns><see langword="true"/> if key was copied. <see langword="false"/> if key was not copied.</returns>
+        /// <remarks>https://redis.io/commands/copy</remarks>
+        bool KeyCopy(RedisKey source, RedisKey destination, int database = 0, bool replace = false, CommandFlags flags = CommandFlags.None);
```

Comment:
a: I would strongly recommend using name like `destinationDatabase` here, to be clear what it means
b: I don't think zero is a good default - because zero is a valid database number; I would *expect* that if I've used `GetDatabase(5)` and then call `.Copy(...)` *without* specifying a database, that we copy it to the *current* database; I see that zero is special-cased in the implementation, but that then means we *can't* intentionally copy to database zero, which is also problematic.

Suggestion: use `-1` as the default value, and interpret any negative number as  "current database"